lib/renderer/security-warnings.js

```diff
@@ -229,23 +229,23 @@ module.exports = {
   },
 
   /**
-   * #10 on the checklist: Do not use blinkFeatures
+   * #10 on the checklist: Do not use enableBlinkFeatures
    *
-   * Logs a warning message about blinkFeatures
+   * Logs a warning message about enableBlinkFeatures
    */
-  warnAboutBlinkFeatures: () => {
+  warnAboutEnableBlinkFeatures: () => {
     const webPreferences = getWebPreferences()
-    if (!webPreferences || !webPreferences.blinkFeatures ||
-        (webPreferences.blinkFeatures.length && webPreferences.blinkFeatures.length === 0)) {
+    if (!webPreferences || !webPreferences.enableBlinkFeatures ||
+        (webPreferences.enableBlinkFeatures.length && webPreferences.enableBlinkFeatures.length === 0)) {
```

Comment:
```js
webPreferences.enableBlinkFeatures.length &&
    webPreferences.enableBlinkFeatures.length === 0
```
This expression can never be evaluated to `true`.
If `length` is there and it's equal to `0` then `.enableBlinkFeatures.length` is treated as `false`.

The whole condition should be written as this:
```js
if (webPreferences === null ||
    !webPreferences.hasOwnProperty('enableBlinkFeatures') ||
    webPreferences.enableBlinkFeatures.length == 0) {
  ...
}
```

Reply:
ah, ok! i'm not sure the thinking behind that when it was originally added but i can definitely update it

Reply:
I guess that's the last thing left for this PR.